docs/concepts/lifecycle.puml

```diff
@@ -1,7 +1,86 @@
 @startuml
-start
-:Draft;
-:Active;
-:Closed;
-stop
+partition #B0E3E6 "Funnel" {
+
+    note right
+      * there is an <b>owner</b>
+      * there is a <b>lead</b> e.g. proposed via owner, BalMATE, ...
+      * the topic has <b>severity</b> via votes
+      ====
```

Comment:
what is the separation for?

Reply:
I use the separator to visually split individual and independent (sequential) actions